script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -33,7 +33,7 @@ Arcueid lets out a cheerful laugh, seemingly amused by something.
 Despite being unresponsive to the antiseptic, it appears that the bandage is able to tickle her.
 <Page10>
 "......"
-I ignore her and fasten the bandage on with a pin. Wounds are said to stop bleeding if you apply pressure, so I give it a final squeeze.
+I ignore her and fasten the bandage on with a pin. Wounds are said to stop bleeding if you apply pressure, so I wrap the final layer of the bandage a little tighter.
```

Comment:
This change isn't really right. 
> 最後はきゅっと力をいれた

This is like... pulling on the ends of the bandage after wrapping it to tighten the whole thing rather than the final layer.
Maybe:
> so I give the ends of the bandages a final tug to pull it tight